ui/app/scripts/stream/views/app-deployed-tooltip.html

```diff
@@ -0,0 +1,32 @@
+<table class="table-hover table-condensed container-details-table">
+    <tbody>
+    <tr>
+        <td class="text-left"><strong>Group</strong></td>
+        <td>{{moduleInstanceData.properties['spring.cloud.application.group']}}</td>
+    </tr>
+    <tr>
+        <td class="text-left"><strong>Host</strong></td>
+        <td>{{moduleInstanceData.properties['spring.cloud.client.hostname']}}</td>
+    </tr>
+    <tr>
+        <td class="text-left"><strong>IP</strong></td>
+        <td>{{moduleInstanceData.properties['spring.cloud.client.ipAddress']}}</td>
+    </tr>
+    <tr>
+        <td class="text-left"><strong>Port</strong></td>
+        <td>{{moduleInstanceData.properties['spring.cloud.client.port'] ? moduleInstanceData.properties['spring.cloud.client.port'] : 8080}}</td>
+    </tr>
+    <tr>
+        <td class="text-left"><strong>Guid</strong></td>
+        <td>{{moduleInstanceData.properties['spring.cloud.application.guid']}}</td>
+    </tr>
+    <tr ng-if="moduleInstanceData.properties['spring.cloud.dataflow.stream.app.type'] !== 'source'">
+        <td class="text-left"><strong>Incoming</strong></td>
+        <td>{{moduleInstanceData.incomingRate.toFixed(8)}}</td>
+    </tr>
+    <tr ng-if="moduleInstanceData.properties['spring.cloud.dataflow.stream.app.type'] !== 'sink'">
+        <td class="text-left"><strong>Outgoing</strong></td>
+        <td>{{moduleInstanceData.outgoingRate.toFixed(8)}}</td>
```

Comment:
Ran into an issue where this is not showing up for me:

![image](https://cloud.githubusercontent.com/assets/685363/25081308/36836572-22e5-11e7-930f-342225f1a561.png)